### PR TITLE
Migrate Volley to Nginx Ingress Controllers

### PR DESCRIPTION
**Pull request by emmadickson**
Supports https://artsyproduct.atlassian.net/browse/PLATFORM-2583

### CODE REVIEW

File: hokusai/production.yml [hunk 1]
```diff
@@ -117,3 +117,39 @@ spec:
...
+metadata:
+  labels:
+    app: volley
+    component: web
+    layer: application
+  name: volley-web-internal
+  namespace: default
+spec:
+  ports:
+    - port: 80
```
Comment (izakp) [inline hokusai/production.yml:132]: Should be 8080 here - application port

File: hokusai/staging.yml [hunk 2]
```diff
@@ -117,3 +117,39 @@ spec:
...
+metadata:
+  name: volley
+spec:
+  rules:
+    - host: imports-staging.artsy.net
+      http:
+        paths:
+          - path: /
+            backend:
+              serviceName: currents-web-internal
```
Comment (izakp) [inline hokusai/staging.yml:154]: s/currents-web-internal/volley-web-internal

File: hokusai/staging.yml [hunk 3]
```diff
@@ -117,3 +117,39 @@ spec:
...
   type: LoadBalancer
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: volley
+    component: web
+    layer: application
+  name: volley-web-internal
```
Comment (izakp) [inline hokusai/staging.yml:128]: s/imports-staging.artsy.net/volley-staging.artsy.net

**Comment by emmadickson** [comment]
@izakp Changes addressed

File: hokusai/production.yml [hunk 4]
```diff
@@ -117,3 +117,39 @@ spec:
...
+    component: web
+    layer: application
+  name: volley-web-internal
+  namespace: default
+spec:
+  ports:
+    - port: 8080
+      protocol: TCP
+      name: http
+      targetPort: 80
```
Comment (izakp) [inline hokusai/production.yml:135]: 8080 here as well

File: hokusai/production.yml [hunk 5]
```diff
@@ -117,3 +117,39 @@ spec:
...
+  selector:
+    app: volley
+    layer: volley
```
Comment (izakp) [inline hokusai/production.yml:138]: `layer: application` here

**Comment by izakp** [comment]
Migrations applied

**Pull request merged**